chef-kitchen/systemd/centos6/Dockerfile

```diff
@@ -11,6 +35,9 @@ RUN yum install -y openssh-server openssh-clients
 # the temporary docker-library:chef_kitchen_systemd_scripts_0.1 is described in this repository
 COPY --from=datadog/docker-library:chef_kitchen_systemd_scripts_0.1 /usr/bin/systemctl /usr/bin/systemctl
 COPY --from=datadog/docker-library:chef_kitchen_systemd_scripts_0.1 /usr/bin/systemctl /usr/bin/systemd
+# replace shebang to use correct version of python
+RUN sed -i 's|#! /usr/bin/python|#! /usr/local/bin/python|g' /usr/bin/systemctl
+run sed -i 's|#! /usr/bin/python|#! /usr/local/bin/python|g' /usr/bin/systemd
```

Comment:
Hack!

Reply:
I agree but the script is vendored and versionned ^^